### PR TITLE
Add stop functionality from old repository psen_scan

### PR DESCRIPTION
**Pull request by hslusarek**

### CODE REVIEW

File: include/psen_scan_v2/scanner_controller.h [hunk 1]
```diff
@@ -64,14 +66,16 @@ class ScannerControllerT
   FRIEND_TEST(ScannerControllerTest, test_start_method_calls_correct_state_machine_event);
   FRIEND_TEST(ScannerControllerTest, test_stop_method_calls_correct_state_machine_event);
   FRIEND_TEST(ScannerControllerTest, test_udp_clients_listen_before_sending_start_request);
+  FRIEND_TEST(ScannerControllerTest, testStopRequestSending);
```
Comment (agutenkunst) [inline include/psen_scan_v2/scanner_controller.h:69]: Try to match the naming scheme to the tests above

File: include/psen_scan_v2/stop_request.h [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  //! @brief Serializes the request into raw data which can be send to the scanner.
+  DynamicSizeRawData toRawData() const;
+
+private:
+  std::ostringstream& processMember(std::ostringstream& os) const;
+  uint32_t calcCrc() const;
+
+private:
+  const std::array<uint8_t, NUM_RESERVED_FIELDS> RESERVED_{};
+  const uint32_t OPCODE_{ htole32(0x36) };
```
Comment (agutenkunst) [inline include/psen_scan_v2/stop_request.h:48]: Do we need htole32?

**Review by agutenkunst — CHANGES_REQUESTED** [review]
Just minor finding. Approve after check on real system.

File: include/psen_scan_v2/scanner_controller.h (hunk 1 above)
Comment (hslusarek) [reply include/psen_scan_v2/scanner_controller.h:69]: Hmm, but I thought we use camel case. Therefore, I suggest changing the existing tests.

File: include/psen_scan_v2/stop_request.h (hunk 2 above)
Comment (hslusarek) [reply include/psen_scan_v2/stop_request.h:48]: StartRequest also uses `htole32()`. But I'm not sure about that...

File: include/psen_scan_v2/scanner_controller.h (hunk 1 above)
Comment (hslusarek) [reply include/psen_scan_v2/scanner_controller.h:69]: Done.

File: include/psen_scan_v2/stop_request.h (hunk 2 above)
Comment (hslusarek) [reply include/psen_scan_v2/stop_request.h:48]: The ensures that  little-endian is send. But maybe we can store the OP_Code as 0x36 and call the function when the `StopRequest` is converter to raw data.

**Pull request merged**